### PR TITLE
promote `format rat` to LE2

### PR DESCRIPTION
**Pull request by StevenClontz**
inspired by #477 and #479

### CODE REVIEW

**Comment by jbunn3** [comment]
I think this looks really nice. I didn't initially go that route, since I didn't know if it would be necessary that early, or if it would be a distraction at that stage. However, since there's already [Activity 1.2.19](https://tbil.org/preview/linear-algebra/LE2.html#LE2-rref-tech), I think it fits nice there with the scaffolding Drew suggested.

**Comment by StevenClontz** [comment]
Exactly, this is the activity I'm adapting. Still a work in progress..

File: source/linear-algebra/source/01-LE/02.ptx [hunk 1]
```diff
@@ -824,66 +824,74 @@ Can <m>\RREF(A)</m> be used to find <m>\RREF(B)</m>?
...
-when RREFs need to be computed.
-    </p>
+            </input>
+        </program>
+    </task>
+    <task>
         <p>
-Try this out to compute
-<m>\RREF\left[\begin{array}{cc|c} 2 &amp; -3 &amp; 1 \\ 3 &amp; 0 &amp; 6 \end{array}\right]</m>.
+Add the line <c>format rat</c> to the top of the cell and re-evaluate
```
Comment (StevenClontz) [inline source/linear-algebra/source/01-LE/02.ptx:867]:
```suggestion
Add the line <c>format rat</c>
(short for <q>format using rational numbers</q>)
to the top of the cell and re-evaluate
```

File: source/linear-algebra/source/01-LE/02.ptx [hunk 2]
```diff
@@ -824,66 +824,76 @@ Can <m>\RREF(A)</m> be used to find <m>\RREF(B)</m>?
...
+Add the line <c>format rat</c>
+(short for <q>format using rational numbers</q>)
+to the top of the cell and re-evaluate
+to see nice fractions in place of the decimal approximations.
         </p>
-    </statement>
+    </task>
+    <task>
+        <p>
+Visit the HTML version of this activity via <c>TBIL.org</c> to use
```
Comment (StevenClontz) [inline source/linear-algebra/source/01-LE/02.ptx:875]:
```suggestion
Visit the HTML version of this activity via <url href="https://TBIL.org/"/> to use
```

File: source/linear-algebra/source/01-LE/02.ptx [hunk 3]
```diff
@@ -824,66 +824,76 @@ Can <m>\RREF(A)</m> be used to find <m>\RREF(B)</m>?
...
+        <p>
+Visit the HTML version of this activity via <url href="https://TBIL.org/"/> to use
+the code cells embedded for your convenience below.
+        </p>
+    </task>
 </activity>
 
 <sage language="octave">
-<input>rref([2,-3,1;3,0,6])</input>
+    <input>rref([1,4,6;2,5,7])</input>
```
Comment (jbunn3) [inline source/linear-algebra/source/01-LE/02.ptx:882]: The only issue here is if they evaluate the next example before this one, then this one will evaluate using format rat. You could put `format none` on the first, but that defeats the purpose of showing them what a normal input would be. I'm not really sure what to suggest here. Maybe we just leave it as is? Alternatively, you could add one last sage insert that has `format none` at the top.

**Review by jbunn3 — COMMENTED** [review]
The rest of the changes look fine, so just take a look at the one comment I made on the format rat part in 02.ptx.

File: source/linear-algebra/source/01-LE/02.ptx (hunk 3 above)
Comment (StevenClontz) [reply source/linear-algebra/source/01-LE/02.ptx:882]:
```suggestion
```

I think the slicker solution is to just remove it; I'm not sure it really adds much value if they already did the one liner above.

File: source/linear-algebra/source/01-LE/02.ptx [hunk 4]
```diff
@@ -824,66 +824,75 @@ Can <m>\RREF(A)</m> be used to find <m>\RREF(B)</m>?
...
+(short for <q>format using rational numbers</q>)
+to the top of the cell and re-evaluate
+to see nice fractions in place of the decimal approximations.
         </p>
-    </statement>
+    </task>
+    <task>
+        <p>
+Visit the HTML version of this activity via <url href="https://TBIL.org/"/> to use
+the code cells embedded for your convenience below.
```
Comment (StevenClontz) [inline source/linear-algebra/source/01-LE/02.ptx:876]:
```suggestion
the code cell embedded for your convenience below.
```

**Review by jbunn3 — APPROVED** [review]
Looks good to me.

**Pull request merged**